app/_src/gateway/kong-enterprise/secrets-management/backends/aws-sm.md

```diff
@@ -88,6 +88,13 @@ In the following example, `AWSCURRENT` refers to the latest secret version and `
 {vault://aws/secret-name/foo#2}
 ```
 
+{:.note}
+> **Note:** The slash symbol (`/`) is a valid character for the secret name in AWS SecretsManager. If you want to reference a secret name that starts with a slash or has two consecutive slashes, transform one of the slashes in the name into URL-encoded format. For example:
+ *  A secret named `/secret/key` should be referenced as `{vault://aws/%2Fsecret/key}`
+ *  A secret named `secret/path//aaa/key` should be referenced as `{vault://aws/secret/path/%2Faaa/key}`
+ Since {{site.base_gateway}} tries to resolve the secret reference as a valid URL, using a slash instead of a URL-encoded slash will result in unexpected secret name fetching.
```

Comment:
```suggestion
>
> Since {{site.base_gateway}} tries to resolve the secret reference as a valid URL, using a slash instead of a URL-encoded slash will result in unexpected secret name fetching.
```
Fixing a whitespace issue that I didn't see before.